autometal-piwik.gemspec

```diff
@@ -18,6 +18,10 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency('xml-simple')
+  # xml-simple needs rexml but v1.1.8 does not specify it in the gemspec
+  # this has been fixed in v1.1.9 but it has not yet been released
+  # so for now we just require it here
+  s.add_dependency('rexml')
```

Comment:
How do we remember to remedy this later?

Reply:
Maybe add a `TODO` in the comments
`# TODO remove dependency when xml-simple has been updated to v1.1.9+`

Reply:
Rather than TODO's it's better to just create an issue

Reply:
I fInd that an issue is overkill.
It is not going to be a problem when it is required both places.
A simple TODO, where the line is should suffice.
Either a TODO or just normal comments.

Reply:
I had a conversation about comments just earlier today with Sergio also haha.
I have no strong preference in this situation. So do what makes most sense

Reply:
I've just created an issue in their repo asking if they will release 1.1.9, maybe they do it soon so I can just use 1.1.9, let's see https://github.com/maik/xml-simple/issues/33